src/pages/patterns/leadspace.mdx

```diff
@@ -5,7 +5,9 @@ description: A lead space is positioned at the top of a web page and serves as t
 
 <PageDescription>
 
-A lead space is positioned at the top of a web page and serves as the first site of viewer engagement; its primary purpose is to orient the user when they land on a page and to help them decide whether to continue reading or not as well as orient themselves. 
+A lead space is positioned at the top of a web page and serves as the first site of viewer engagement.
+
+Its primary purpose is to orient the user when they land on a page, inform them of the content of the page, and guide them to the first key content, whether it’s a piece of content or a CTA button.
```

Comment:
Its primary purpose is to orient the user when they land on a page, inform them of the content, and guide them to the first key piece of content on the page, whether it's a crucial piece of information or a call-to-action button.